src/vsr/grid_scrubber.zig

```diff
@@ -156,6 +159,71 @@ pub fn GridScrubberType(comptime Forest: type) type {
             scrubber.* = undefined;
         }
 
+        pub fn open(scrubber: *GridScrubber, random: std.rand.Random) void {
+            // Compute the tour origin exactly once.
+            if (scrubber.tour_tables_origin) |_| {
```

Comment:
```suggestion
            if (scrubber.tour_tables_origin != null) {
```

this looks slighly less surprising to me, though either way works